backend/app/controllers/projects.js

```diff
@@ -437,9 +437,14 @@ module.exports = {
                                         .and(Task.uoaId.equals(productUoas[p].UOAid))
                                 )
                             );
-                            usersWithLiveTasks.push(_.first(tasks)); // Push user ids to list
```

Comment:
Take a look on line 434 above. Basically, the order of those tasks is coming out by the rows of the UOAs, and the order isn't guaranteed because the user can assign tasks in any order they choose. What you need to do is parse over EACH task in this array and find those with a `startDate` greater than today.